github-actions/feature-request/src/action.ts

```diff
@@ -82,6 +85,8 @@ export const run = async (api: GitHubAPI, config: Config) => {
 const processIssue = async (githubAPI: GitHubAPI, githubIssue: GitHubIssueAPI, config: Config) => {
   const issue = await githubIssue.get();
 
+  log(`Started processing issue #${issue.number}`);
```

Comment:
I think it might make sense here to provide the metadata of the issue for ease of understanding the issue.

```suggestion
  log(
    `Started processing issue #${issue.number}:\n` +
    `  Title:    ${issue.title}\n` +
    `  Author:   ${issue.author}\n` +
    `  Votes:    ${issue.reactions['+1']}\n` +
    `  Comments: ${issue.numComments}`
    );
```